compiler/entry/jasmin2ec.ml

```diff
@@ -59,7 +59,7 @@ let model =
     $(b,CT): Functions additionally return timing-observable leakage for
     'cryptographic constant time' (if/while conditions, memory access
     addresses, array indices, for loop bounds).
-    $(b,CTG): Cryptographic constant time leakage is added to a
+    (Deprecated) $(b,CTG): Cryptographic constant time leakage is added to a
```

Comment:
As far as I understand, `doc` is for one-line description of an argument. You should use `man` to give a detailed explanation (and start a new paragraph for each possible argument to have a nice-looking man page). But this orthogonal to this PR.

Reply:
The [`info`](https://erratique.ch/software/cmdliner/doc/Cmdliner/Arg/index.html#val-info) value does not accept any `~man` argument, does it?

Reply:
Indeed, sorry, I was confused by the `info` function in `Cmd`.